stl/team_aster/wangzhongren/learning.md

```diff
@@ -1 +1,38 @@
-it's learning document.
+## The learning record of stl_list.
+
+### 简介
+
+list 是一个双向环形链表，只需要一个指针，便可完整表现整个链表；在list中任何位置插入和删除的速度都很快。
```

Comment:
It would be a nice idea to show us the time/space complexity of their various operations